cms/static/js/features_jsx/studio/index.jsx

```diff
@@ -0,0 +1,102 @@
+import React, {Component} from 'react';
```

Comment:
Actually, I don't think eslint is running against this file at all, since it doesn't yet know to look at `.jsx` files. ESLint determines which filetypes to lint based on `--ext` parameters [passed on the command line](http://eslint.org/docs/user-guide/configuring#specifying-file-extensions-to-lint) which fortunately is pretty straightforward to fix. Within our `run_eslint` quality task, just add `--ext .js --ext .jsx` before `--format-compact` on this line: https://github.com/edx/edx-platform/blob/7bc87a5f5672e65a01034ed850aeb43f8813ef3a/pavelib/quality.py#L291.